notebooks/week2/02_04_train_log_custom_model.py

```
@@ -0,0 +1,169 @@
+
+# COMMAND ----------
+model_version_alias = "the_best_model"
+client.set_registered_model_alias(model_name, model_version_alias, "1")  
+ 
+model_uri = f"models:/{model_name}@{model_version_alias}"
+model = mlflow.pyfunc.load_model(model_uri)
+
+# COMMAND ----------
+client.get_model_version_by_alias(model_name, model_version_alias)
+# COMMAND ----------
+model
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)